CHANGELOG.md

```diff
@@ -85,11 +103,12 @@
 
 ### Fixes
 
+- Improved error handling for incorrect passphrase and incorrect hardware wallet error ([PR 2860](https://github.com/input-output-hk/daedalus/pull/2860))
```

Comment:
Should this be on fixes of 4.11.0?

Reply:
Hmmm… Failed automatic conflict resolution in this merge? https://github.com/input-output-hk/daedalus/commit/46c38d6ec0b095d1e1f695c5b52de3bb3b08ad27#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR103

The only conflicts there were in 
```
# Conflicts:
#	CHANGELOG.md
```

Look at the diff of that PR, it’s wrong as well → https://github.com/input-output-hk/daedalus/pull/2948/files

Reply:
Yeah, it was possibly included wrong originaly https://github.com/input-output-hk/daedalus/pull/2860/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR88

Reply:
Soooo… I think this should go into the 4.10.0 section, right?

Reply:
Not sure, because this task is on the 4.11.0 release doc